dev-libs/botan/botan-2.0.1.recipe

```diff
@@ -0,0 +1,71 @@
+SUMMARY="A cryptography library written in C++"
+DESCRIPTION="This is the best option for crypthography in C++"
+HOMEPAGE="https://botan.randombit.net/"
+COPYRIGHT="2000-2017 botan"
```

Comment:
Well, according to the authors.txt file, Jack Lloyd is the only author. The next 6 people have just made "extensive and ongoing contributions", so my understanding is that when they contribute they give their copyright to "The Botan Authors"... which is currently Jack Lloyd.

But I guess your suggestion is OK.
BTW, do we squeeze "1999-2013,2014,2015,2016,2017" to "1999-2017"?